mmv1/templates/terraform/decoders/bigquery_analyticshub_listing_subscription.go.erb

```diff
@@ -0,0 +1,15 @@
+//name := d.Get("name").(string)
+
+res["destination_dataset"] = map[string]interface{}{
```

Comment:
These keys should be camelCased instead of snake_cased! If you look at https://github.com/modular-magician/terraform-provider-google-beta/compare/modular-magician:8d690f3...modular-magician:a484d5b#diff-1848eef73aff2900a034768229b6badf2e95f44b283c2de46883f7b9777ebeadR305 the flattener is looking for `destinationDataset` within `res`, the decoder transforms the response message but doesn't change the casing of the field names as that's done later.

Reply:
(@NickElliot I'm responding because of a YAQS!)

Reply:
AHHHH!!! Thank you! I'll try it out now :)